activemq-broker/src/main/java/org/apache/activemq/broker/BrokerService.java

```diff
@@ -153,9 +153,7 @@
     private boolean useLoggingForShutdownErrors;
     private boolean shutdownOnMasterFailure;
     private boolean shutdownOnSlaveFailure;
-    private boolean waitForSlave;
     private long waitForSlaveTimeout = DEFAULT_START_TIMEOUT;
```

Comment:
If waitForSlave can be removed, can waitForSlaveTimeout also be removed?

Reply:
I thiiiink so. I recall going through all the properties, but maybe I missed this one because the name was so similar - I can't find any references to it in the codebase or where an embedded broker might use it.